File: ALGORITMOS DE PROGRMACION/Sesion 16/Tarea/tarea.py
import math as m 
e=m.exp(1)
x = int (input("x:"))
n = int (input("n:"))

suma = 0 
i = 0 

while i <= n:
    facti = 1
    j=2
    while j <= i:
        facti *= j
        j += 1
        print(i, "! = ", facti)
        ponxi  =  1
        k = 1
        while k <= i:
            ponxi *= x
            k += 1
            print(x, "a la ", i, " = ", ponxi)
            i += 1 
            suma = ponxi / facti
            print(e, "a la ", x , "es igual a ", m.exp(x))
            print("Su aproximacion es igual a ", suma)

File: ALGORITMOS DE PROGRMACION/Sesion 16/Tarea/tarea.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {

double  e = exp(1);
int x, n;

cout << "x:"
cin >> x;

cout << "n:"
cin >> n;

double suma = 0 ;

for (int i = 0; i <= n; ++i) {
    double facti = 1;
    for (int j = 2; j <= i; ++j) {
        facti *= j;
    }
    cout << "! = " << facti << endl;

    double ponxi = 1;
    for (int k = 1; k <= i; ++k) {
        ponxi *=x;
    }
    cout << x << " a la " << i << " = " << ponxi << endl ;
    suma += ponxi / facti;

}

cout << "e a la " << x << " es igual a " << exp(x) << endl;
cout << "Su aproximacion es igual a " << suma << endl;

return 0;





}